README.md

```diff
@@ -1,3 +1,127 @@
-# field_monitoring
+![](RackMultipart20220705-1-ruigrg_html_70e3b217e6f91606.png)
+
+# ATLAS Service Template Specification
+
+# Name: Crop Status Monitoring
+
+# Author(s): AgroApps (AGA)
+
+_The &quot;__Crop Status Monitoring&quot; ATLAS Service Template_
+
+This service will deliver a number of EO derived indicators for crop status monitoring consisting of:
+
+- Vegetation Indices
+
+NDVI
+
+- Biophysical Parameters for annual crops
+
+Biomass
+
+LAI
+
+- Chl Content
+- Canopy Water Content
+
+and a number of farm services, such as:
+
+- Yield Estimation
+- Crop Residues
+
+The system for the calculation and application of the VIs will be optimized with respect to crop types. To ensure high temporal coverage of the data, the system will utilize data from optical satellite sensors, namely the Sentinel-2.
```

Comment:
This paragraph is implementation specific and should not be included as a service template requirement.